docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -164,3 +164,257 @@ The test results indicate that the minimum amount of memory for running {Product
 ** Installing the cluster monitoring tools requires an additional 5 GB of memory.
 ====
 
+== Red Hat Single Sign-On
+{ProductShortName} creates a basic
+https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6[Red
+Hat Single Sign-On] instance, using the Keycloak CR of Red Hat Single
+Sign-On Operator. Keycloak is a single sign-on solution for web apps and
+RESTful web services.
+
+The {ProductShortName} Keycloak realm is fully configured by the Hub.
+
+{ProductShortName} does not enable RHSSO external access by default. However, if you
+require RHSSO, you can configure it by using the Tackle CR attribute
+`rhsso_external_access`. Otherwise, use the {ProductShortName} realm administration console to
+adjust the settings.
+
+For more information, see
+https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6/html/server_administration_guide/red_hat_single_sign_on_features_and_concepts[Red
+Hat Single Sign-On features and concepts].
+
+=== Roles and Permissions
+
+The following table contains the roles and permissions (scopes) that {ProductShortName}
+seeds the RHSSO with:
+
+[width="100%",cols="34%,33%,33%",]
+|===
+|*tackle-admin* |*Resource Name* |*Verbs*
```

Comment:
@jortel can we confirm these are the role names we are using downstream as well?

Reply:
I don't know how to confirm downsteam.
@fbladilo Did you edit the [roles.yaml](https://github.com/konveyor/tackle2-hub/blob/main/auth/roles.yaml) downstream? If not, they are the same.